spec/queries/find_exact_query_spec.rb

```diff
@@ -46,6 +46,12 @@
       _subscriber_list = create_subscriber_list(document_type: 'travel_advice')
       expect(query.exact_match).to be_nil
     end
+
+    it "not matched on content_id - even if they match" do
+      query = build_query(links: { policies: { any: %w[aa-11] } }, document_type: 'cd4179c9-6fe4-49bc-8790-da8a1406c8c5')
```

Comment:
not sure I fully understand this test - should `document_type` be `content_id`? (similar question for the other tests you've added in this commit)

Reply:
Hmm you're right this a bit unclear. I think I just wanted a random string for the document type so I generated a UUID which then makes it look like a content ID, and I also forgot to rename the tests.